src/utils/Firebase.tsx

```diff
@@ -42,10 +42,34 @@ export async function getUserDataForSpecificTask(jobID: string) {
     console.log(userDoc.data());
 }
 
+export async function createNewJob(userID: string, title: string, description: string, jobSkills: string[]) {
+    try {
+        const newJob = {
+            userID: userID,
+            title: title,
+            description: description,
+            jobSkills: jobSkills,
+            createdAt: new Date()
+        };
+        
+        const docRef = await addDoc(collection(db, "Jobs"), newJob);
+        const returnedJob = await getDoc(docRef);
```

Comment:
Do we really need this? Wouldn't a successful docRef guarantee that this write is successful?